components/annot-gender.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+<div>
+  <annot-explanation :explanation="explanation"></annot-explanation>
+</div>
+</template>
+
+<script>
+export default {
+  name: "annotate_gender",
```

Comment:
Let's now be consistent with the term 'sex' or 'gender'. What is the consistent usage 
in the neuroimaging community? What is typically coded? Is 'gender' a term that's actually 
being coded for or is it actually biological sex?

Reply:
I think that's a good question. I think "sex" is the general standard and also [used in BIDS](https://bids-specification.readthedocs.io/en/stable/99-appendices/14-glossary.html#sex-columns), but I think that in the vast majority of cases, what we encode with this is the self-reported gender of participants. That said, I think this is something we shouldn't just decide on our own. And so for now, it might make sense to go with "sex" everywhere and keep this in mind for a future revision.

Reply:
Sounds good.